src/config.rs

```diff
@@ -2406,7 +2410,9 @@ impl TiKvConfig {
             }
         }
 
-        if last_cfg.raft_store.raftdb_path != self.raft_store.raftdb_path {
+        if !self.raft_engine.enable
```

Comment:
`raftdb_path` is still need to be checked, because maybe it's necessary to dump from it.